### DISCUSSION TITLE
base64 audio data

### DISCUSSION THREAD

**Discussion by acn-reginald-casela** [ggml-org/whisper.cpp#2093] [Q&A] [answered]
Is there a way to use base64 data instead of passing over a wav file?

I plan to deploy this as a server and , with the way the front-end payload is setup, my expected input is in base64 format.

**Comment by ulatekh** [comment, marked as answer]
What is your question, exactly? Are you looking for [this](https://stackoverflow.com/questions/180947)?

**Comment by patelaryan0914** [comment]
> Is there a way to use base64 data instead of passing over a wav file?
> 
> I plan to deploy this as a server and , with the way the front-end payload is setup, my expected input is in base64 format.

have you figured it out?
